### PR TITLE
Add Talanta Mtaani project

### PR DESCRIPTION
**Pull request by Nemwel-Boniface**
## Hello, @ahangarha :wave: :wave:!   
I am enjoying contributing to your open-source project. Below I will document the changes I have made and my proposals to make the project better:

## 1. Included my project 
I have included my module 1 capstone project, Talanta Mtaani, among the list of projects that are to be featured. I have included : 

1. Project Author
2. Project name
3. Project description
4. Publish date
5. My GitHub username

## 2. :recycle: Suggested changes to the project styling :recycle: 
- The project runs perfectly well on a desktop version. However, not all users will be viewing the project from a desktop and might be having a smaller screen size for their mobile phones.
- This is how I saw it looking on mobile phone view. The featured projects would be better if they stacked each on top of the other and not look squeezed to give your users a better user experience when using the project.
MVprojects

I am not familiar with Tailwind CSS and was unable to make that suggested change. 
I am looking forward to my project being featured :smiley:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,12 +5,13 @@ Contribution to this project is very much like contributing to any other Free/Li
 3. Make a new branch from `develop` branch (This is important here! You need to checkout to `develop` branch and then make a branch)
 4. Make the local development environment ready by running `npm install`.
 5. To run the project on live server, run `npm run watch`.
-6. Commit your changes.
-7. Push your commits to your own fork
-8. Send Pull Request from your new branch to this project's **develop branch**.
+6. To run the project on local host, run `npm run build` then `npm start`.
+7. Commit your changes.
+8. Push your commits to your own fork
+9. Send Pull Request from your new branch to this project's **develop branch**.
```
Comment (ahangarha) [inline CONTRIBUTING.md]: Please undo these changes and send them as a separate PR.

Please try to keep PR and its commits all relevant.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -5,12 +5,13 @@ Contribution to this project is very much like contributing to any other Free/Li
...
 
 You are done. I will get notified and then take the next steps accordingly.
 
 If later you need to contribute, you need to first make sure your fork is up to date. You should see a button on your fork for synchronization. After that, you need to use `git pull` on your clone (on you local machine) to get latest commits and then the rest is the same...
 
-This project is under AGPL license. 
+This project is under AGPL license.
```
Comment (ahangarha) [inline CONTRIBUTING.md:16]: Please send this as a separate PR

**Review by ahangarha — CHANGES_REQUESTED** [review]
Thanks for your PR. :white_flower: 

I commented on some of the changes. Please follow those instructions and then let me know.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Nemwel-Boniface) [reply CONTRIBUTING.md:16]: Hello @ahangarha ,
I have made the requested changes to this pull request. Thank you.

Comment (ahangarha) [reply CONTRIBUTING.md:16]: Still this line shouldn't be included in this PR. This is not related to your own project. Please revert this change in this PR

Comment (ahangarha) [reply CONTRIBUTING.md:16]: I decided to ignore this little irrelevant issue and proceed for merge.

Please keep in mind to keep your commits in PR all relevant to the PR topic.

Thanks again.

**Review by ahangarha — APPROVED** [review]
Thank you
